Lib/test/test_string/_support.py

```diff
@@ -1,6 +1,31 @@
+import unittest
 from string.templatelib import Interpolation
 
 
+class TStringTestCase(unittest.TestCase):
+    def assertTStringEqual(self, t, strings, interpolations):
+        """Test template string literal equality.
+
+        The *strings* argument must be a tuple of strings equal to *t.strings*.
+
+        The *interpolations* argument must be a sequence of tuples which are
+        compared against *t.interpolations*. Each tuple consists of
+        (value, expression, conversion, format_spec), though the final two
+        items may be omitted, and are assumed to be None and '' respectively.
+        """
+        self.assertEqual(t.strings, strings)
+        self.assertEqual(len(t.interpolations), len(interpolations))
+
+        for actual, expected in zip(t.interpolations, interpolations,
+                                    strict=True):
+            if len(expected) == 2:
+                self.assertEqual(tuple(actual), expected + (None, ''))
```

Comment:
I don't think the `tuple(actual)` call will work since `Interpolation` is not an iterable.

Reply:
Of course, sorry -- I wrote this on a computer without a working build of the branch. I've pushed a fix.